one-ui/ui/api/swagger/mocks.yaml

```diff
@@ -2275,6 +2275,39 @@ definitions:
                 - '{}'
                 - '{}'
             version: 0
+    runFlowResponse:
```

Comment:
This is interesting, as I was thinking we should define a JSON Schema for objects like this that are returned by key classes. Do you know if stepResponse is expected to be defined? That has some pretty key information in it. 

I think it's fine to leave this here, just makes me wonder what's a good way to define this stuff in one place.

Reply:
It can be derived by the code. It is a little different than other definitions in that the property names are dynamic based on what steps are run and if they provide a response. I'm not sure if and/or how properties like that are defined in swagger yet. Perhaps I just start by adding properties "1" and "2" while noting that they are dynamic based on the steps run?